docs/dev/news/3970.feat

```diff
@@ -0,0 +1,5 @@
+Enable finer options for deciding what to verify. Users can now choose to verify: only root set files; also files reached through include directives, or everything including files from libraries and .doo files.
+
+To prevent users from accidentally using unverified library files, `--verify-scope=RootSourcesAndIncludes`, previously `--verify-included-files`, now requires any files passed to `--library` to contain a marker that they have been verified previously, such as is present in `.doo` files. `--verify-scope=RootSources` does not have this requirement, since it already allows skipping the verification of files included through directives.
```

Comment:
> I'd simplify this to just say .doo files as the only way to indicate that, since there's no other kind of supported marker.

Don't `dlls` work as well?

Reply:
> Don't dlls work as well?

True I keep forgetting about that since no-one is using that feature. :)

Reply:
> This explanation helps understand why we made the change and is a good precursor to the migration guide which belongs in this release note. What's the downside of a large release note?

Fair enough, no objection!